backends/p4tools/testgen/targets/bmv2/program_info.h

```diff
@@ -1,5 +1,5 @@
-#ifndef BACKENDS_P4TOOLS_TESTGEN_TARGETS_BMV2_PROGRAM_INFO_H_
-#define BACKENDS_P4TOOLS_TESTGEN_TARGETS_BMV2_PROGRAM_INFO_H_
+#ifndef TESTGEN_TARGETS_BMV2_PROGRAM_INFO_H_
```

Comment:
This an incorrect header guard definition. I wonder why cpplint does not catch this.